cmd/build-kube.go

```diff
@@ -112,6 +118,20 @@ func init() {
 		"Factor for determining limits from requests",
 	)
 
+	buildKubeCmd.PersistentFlags().BoolP(
+		"use-cpu-limits",
+		"",
+		true,
+		"Include cpu limits when generating helm chart",
+	)
+
+	buildKubeCmd.PersistentFlags().IntP(
+		"cpu-limit-factor",
+		"",
+		2,
+		"Factor for determining limits from requests",
```

Comment:
Can you please improve this description? I don't understand it as-is.

Reply:
This is currently an integer number used to calculate a limit from the request, i.e
`limit = factor * request` ... Analogous to memory req/limit, except feeling somewhat wrong and not having anything else.